lib/localeapp/cli/install.rb

```diff
@@ -2,35 +2,39 @@ module Localeapp
   module CLI
     class Install < Command
       attr_accessor :config_type
+      attr_accessor :write_dotenv
 
       def initialize(args = {})
         super
         @config_type = :default
+        @write_dotenv = false
       end
 
       def execute(key = nil)
-        installer("#{config_type.to_s.capitalize}Installer").execute(key)
+        installer("#{config_type.to_s.capitalize}Installer").execute(key, write_dotenv)
```

Comment:
I would favor a keyword argument here.
Passing a new boolean dependent on position, for each new option the
installer might require is not easy to maintain.